hypha/static_src/src/app/src/containers/SubmissionFilters/index.js

```diff
@@ -167,7 +167,7 @@ export class SubmissionFiltersContainer extends React.PureComponent {
                       <a 
                       style={{color: "black"}}
                       target="_blank" rel="noreferrer" 
-                      href={"/apply/submissions/grouped-applications/?id="+ Object.keys(this.props.submissions).toString()}
+                      href={"/apply/submissions/summary/?id="+ Object.keys(this.props.submissions).toString()}
```

Comment:
Now the "Share this filter" goes to the "Application summary list". In the screencast (really like that you add them to the PR!) you get a notice that the URL is copied to the clipboard.

Reply:
@frjo I couldn't reproduce this issue, can you share the link of test environment?

Kindly [clear the cache](https://superuser.com/a/512833/454365) and try once

Reply:
@vimal1083  This is how the link looks to my browsers `<a target="_blank" rel="noreferrer" href="/apply/submissions/summary/?id=223" style="color: black;">Share this filter</a>`.

Reply:
I have deployed this to test now and there the copy to clipboard works. Something wrong on my local setup.

But I'm seeing another error when filtering larger sets of applications. https://reactjs.org/docs/error-decoder.html/?invariant=185

"Maximum update depth exceeded. This can happen when a component repeatedly calls setState inside componentWillUpdate or componentDidUpdate. React limits the number of nested updates to prevent infinite loops."

Try to filter for something here e.g. https://test-apply.opentech.fund/apply/submissions/all/?submission=19

Reply:
@vimal1083 Also, can we make the "URL copied to clipboard" message disappear automatically after a few seconds?

Reply:
Seems I'm getting a cross origin error when I test this in my local setup, see below.

```
TypeError: undefined is not an object (evaluating 'navigator.clipboard.writeText')
Error: A cross-origin error was thrown. React doesn't have access to the actual error object in 
```

I see that `'Access-Control-Allow-Origin': '*',` is set in `webpack.dev.config.js`. Any ideas?

Reply:
Added a timeout for notification.

You get the above error not because of  'Access-Control-Allow-Origin': '*'. I guess it is because your'e accessing app in `https` without a proper certificate, try to open the app in `http`. 

If you get to see the same error in staging environment , LMK